docs/content/en/blog/articles/release-1.0/index.md

```diff
@@ -0,0 +1,102 @@
+---
+date: 2021-02-26
+title: "K8ssandra 1.0 Stable Release and What's Next"
+linkTitle: "K8ssandra 1.0 Release"
+description: >
+  The K8ssandra 1.0 release is a significant milestone for the project, reflecting months of work and contributions from the community. This release represents a production-ready version of the project and sets the stage for continued development.
+author: Chris Bradford ([@bradfordcp](https://twitter.com/bradfordcp))
+
+---
+
+A project's 1.0 release is a significant milestone. In some cases it is the
+solidification of an API. In others, it’s a reflection of its suitability for
+production workloads, and sometimes it’s just because the developers like round
+numbers. For K8ssandra, the 1.0 release represents a version that is ready for
+widespread adoption. That means our 1.0 release provides configuration
+interfaces with smart, well-tested defaults. It also means shipping thoughtful
+documentation to guide you through deployment and operation. Beyond the code
+bits and Helm charts, the project has grown significantly. We've seen an
+evolution of how features are developed, validated, and deployed. Automation has
+been added to drive some of these processes and ease the development workflow.
+
+The K8ssandra team has shipped regular updates to the Helm repository, each one
+reflecting new functionality, bug fixes, and enhancements. Part of a 1.0 release
+is a confidence in stability and measured release of major, minor, and patch
+level changes. K8ssandra 1.0 also brings online a pair of release streams:
+Stable and Next. The Stable stream contains releases that have undergone
+additional testing and validation prior to being published. Alternatively, users
+looking for the latest in-flight functionality may leverage the Next stream.
+This new stream exposes chart updates as they are merged into the main branch.
+Whether you are looking to take the latest feature for a spin, or validate a
+bugfix before a wider release, you can do so without having to clone the repo
+and build from source.
+
+Since the initial release of K8ssandra at Kubecon NA 2020, there have been 55
```

Comment:
I counted. 59! As of Kubecon, we were at 0.10.0. We've mostly only incremented that number, but there has also been 0.17.1, 0.27.1, 0.42.1, 0.47.1, 0.49.1, 0.60.1, 0.60.2, and finally 0.60.3, plus of course 1.0.0 itself.

```suggestion
Since the initial release of K8ssandra at Kubecon NA 2020, there have been 59
```